src/Client/Pages/Content/Home.razor

```diff
@@ -28,7 +28,7 @@
         <MudItem xs="12" sm="12" md="12">
             <MudExpansionPanels>
                 <MudExpansionPanel Text="@_localizer["Getting Started"]" MaxHeight="150">
-                    @_localizer["Increase you are stuck anywhere or have any queries regarding this implementation, I have compiled a Quick Start Guide for you reference."]
+                    @_localizer["Encase you are stuck anywhere or have any queries regarding this implementation, I have compiled a Quick Start Guide for you reference."]
```

Comment:
typo again :p Could you change it to 'Incase' and update the Pull request. Thanks